src/Algebra/Graph/AdjacencyIntMap/Algorithm.hs

```diff
@@ -113,7 +170,7 @@ dfs vs = concatMap flatten . dfsForestFrom vs
 -- 'isSubgraphOf' ('vertices' $ reachable x y) y == True
 -- @
 reachable :: Int -> AdjacencyIntMap -> [Int]
-reachable x = dfs [x]
+reachable x = concat . bfs [x]
```

Comment:
Please update the comment, the examples, as well as the corresponding tests to match the new implementation.